types/gapi.client.classroom/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for non-npm package Google Google Classroom API v1 1.0
+// Type definitions for non-npm package Google Classroom API v1 1.0
 // Project: https://developers.google.com/classroom/
-// Definitions by: Bolisov Alexey <https://github.com/Bolisov>
+// Definitions by: Maxim Mazurok <https://github.com/Maxim-Mazurok>
```

Comment:
I don't usually like to see authors removed except by themselves. Does @Bolisov no longer want to see updates?

Reply:
They were pinged much earlier on by TS bot here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40407#issuecomment-554205819

They were the original authors of the type generator which we've forked since they've gone unresponsive. Their profile also indicates that they haven't had any Github activity [since 2018](https://github.com/Bolisov) ¯\\\_(ツ)_/¯